README.md

```diff
@@ -41,7 +41,7 @@ You **need to install Qt as described below** instead of using pre-built package
     * Windows: Make sure to install VS 2015 32 bit package.
 
 ###### Install additional packages:
-* Ubuntu: sudo apt-get install speech-dispatcher libudev-dev libsdl2-dev
+* Ubuntu: sudo apt-get install speech-dispatcher libudev-dev libsdl2-dev libgstreamer1.0-0 gstreamer1.0-plugins-base libgstreamer-plugins-base1.0-dev gstreamer1.0*
```

Comment:
I assume similar packages will not be needed for Fedora, archilinux, windows?

Reply:
I didn't try on those. I tried in ubuntu which gave me some build errors and found out that these packages were also needed. If I later try with windows, I'll update this document accordingly.

Reply:
Works for me. Added @DonLakeFlyer as reviewer :-)